example/src/main/java/io/honeycomb/opentelemetry/android/example/MainActivity.kt

```diff
@@ -47,8 +51,11 @@ enum class PlaygroundTab(
     CORE("Core", Icons.Outlined.Home, Icons.Filled.Home),
     UI("UI", Icons.Outlined.Palette, Icons.Filled.Palette),
     NETWORK("Network", Icons.Outlined.Language, Icons.Filled.Language),
+    VIEW_INSTRUMENTATION("Render", Icons.Outlined.Straighten, Icons.Filled.Straighten),
 }
 
+val LocalOtelComposition = compositionLocalOf<OpenTelemetryRum?> { null }
```

Comment:
I saw that the `CorePlayground` accepts the `OpenTelemetryRum` object as a parameter but I didn't want to pass it through the many layers of nested Composables in the `ViewInstrumentationPlayground`, and I thought it was easier for `HoneycombInstrumentedComposable` to read from a context-like thing instead of accepting an `OpenTelemetryRum` object.

I'm open to push this back to a parameter if we think that's a better idea, though.

Reply:
I think this makes a lot of sense, and fits the criteria for when to use it as described on https://developer.android.com/develop/ui/compose/compositionlocal#intro

But this would effectively become part of our public API at that point, right? Our developers would have to use this line?
`CompositionLocalProvider(LocalOtelComposition provides otelRum) {`

If so, we should think carefully about how to name `LocalOtelComposition`.

Reply:
Yeah, that's true. IntelliJ reports that the convention here is that the key starts with the `Local` prefix:

![image](https://github.com/user-attachments/assets/787f4b4b-f018-4248-ac0e-ee5d011f0753)

The CompositionLocalProvider pattern reminds me of React Context. In that ecosystem, it's common for libraries to export their own Providers and consumers use that as an entry point. ex:

```kotlin
// MainActivity.kt

setContent {
  HoneycombLocalProvider( /* honeycomb config */ ) {
    // body content
  }
}
```

and
```kotlin
// HoneycombLocalProvider.kt

val LocalOpenTelemetryRum = compositionLocalOf<OpenTelemetryRum?> { null }

@Composable
fun HoneycombLocalProvder( /* config */ ) {
  val options = HoneycombOptions.builder(this)
    // config
    .build()

  otelRum = Honeycomb.configure(this, options)

  CompositionLocalProvider(LocalOpenTelemetryRum provides otelRum) {
    // child content
  }
}
```